Instructor-Guide--Assignments--Setting-Up.md

```diff
@@ -35,8 +35,13 @@ This section allows you to set the name and due date for the assignment as well
 
 **6. Check boxes:** The rest of this section includes check boxes that may be selected or deselected depending on your preferences. Note that in order for students to submit online, the "Allow students to submit through the web interface" box must be checked. If you prefer students to submit through a version control system, then uncheck the "web interface" box and select "version control system".
 
+If you wish to allow students to submit URLs check the "Allow students to submit URLs" box. This is especially useful if you plan on requiring students to submit videos or large files since preview support is available for content from YouTube and Google Drive/Docs.
+
 Please see the [Peer Review](Instructor-Guide--Assignments--Peer-Review.md) page for more information on Peer Review.
 
+> :spiral_notepad: **NOTE:**
```

Comment:
@anoverga17 Move this to above the previous sentence. It is much more important to put notes next to the text they're related to.